mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala

```diff
@@ -91,26 +142,29 @@ class DefaultSource extends RelationProvider with SchemaRelationProvider with Cr
 
     val readpreference = parameters.getOrElse(readPreference, DefaultReadPreference)
 
-    val properties :Map[String, Any] =
-      Map(Host -> host, Database -> database, Collection -> collection , SamplingRatio -> samplingRatio, readPreference -> readpreference)
+    val properties: Map[String, Any] =
+      Map(Host -> host, Database -> database, Collection -> collection, SamplingRatio -> samplingRatio, readPreference -> readpreference)
 
-    val optionalProperties: List[String] = List(Credentials,SSLOptions, IdField, SearchFields, Language, Timeout)
+    val optionalProperties: List[String] = List(Credentials, SSLOptions, IdField, SearchFields, Language, Timeout)
+
+    val finalMap = (properties /: optionalProperties) {
+      //TODO improve code
+      case (properties, Credentials) =>
 
-    val finalMap = (properties /: optionalProperties){    //TODO improve code
-      case (properties,Credentials) =>
         /** We will assume credentials are provided like 'user,database,password;user,database,password;...' */
         val credentialInput = parameters.getOrElse(Credentials, " ")
-        if(credentialInput.compareTo(" ")!=0){
-          val credentials= credentialInput
+        if (credentialInput.compareTo(" ") != 0) {
+          val credentials = credentialInput
             .split(";")
             .map(credential => credential.split(",")).toList
             .map(credentials => MongodbCredentials(credentials(0), credentials(1), credentials(2).toCharArray))
           properties.+(Credentials -> credentials)
         } else properties
-      case (properties,SSLOptions) =>
+      case (properties, SSLOptions) =>
+
         /** We will assume ssloptions are provided like '/path/keystorefile,keystorepassword,/path/truststorefile,truststorepassword' */
         val ssloptionInput = parameters.getOrElse(SSLOptions, " ")
-        if(ssloptionInput.compareTo(" ")!=0) {
+        if (ssloptionInput.compareTo(" ") != 0) {
```

Comment:
It's not my code. I proposed to use isEmpty but we are going to replace with the new Mongo code as soon as posible. Anyway, it's not related to this PR.